### DISCUSSION TITLE
自己训练了两个模型，能否合并成一个模型，无需重复训练

### DISCUSSION THREAD

**Discussion by yuchangqing** [ultralytics/yolov5#7960] [Q&A] [answered]
问题1.两个训练好的模型，是否可以合并成一个模型
问题2.我有一个已训练好的模型，如何在此基础上如何增加数据集进行训练
先感谢各位大神，帮忙解答

**Comment by glenn-jocher** [comment, marked as answer]
See https://community.ultralytics.com/t/how-to-combine-weights-to-detect-from-multiple-datasets
